app/validators/base_path_validator.rb

```diff
@@ -0,0 +1,11 @@
+class BasePathValidator < ActiveModel::Validator
+  def validate(record)
+    unless record.mutable_base_path?
+      live_item = LiveContentItem.find_by(content_id: record.content_id)
+
+      if live_item.present? && live_item.base_path != record.base_path
+        record.errors.add(:base_path, 'is immutable for published items')
```

Comment:
At some point this message will be exposed to the user, so we probably want to avoid words like "immutable". Perhaps "cannot be changed for published items"?